spec/counter_culture_spec.rb

```diff
@@ -2304,6 +2304,31 @@ def mess_up_counts
     expect(prefecture.reload.big_cities_count).to eq(1)
   end
 
+  it "support fix counts using batch limits start and finish" do
+    company = Company.create!
+    company.children << Company.create!
+    company.children_count = -1
+    company.save!
+
+    companies = 3.times.map do
+      company = Company.create!
+      company.children << Company.create!
+      company.children_count = -1
+      company.save!
+      company
+    end
+
+    start = companies.first.id
+    finish = companies.last.id
+
+    fixed = Company.counter_culture_fix_counts start: start, finish: finish
+    expect(fixed.length).to eq(3)
+
+    companies.each do |company|
+      expect(company.reload.children_count).to eq(1)
+    end
```

Comment:
Should we also test that `company` _wasn't_ fixed here? To make sure that it's only touching the IDs that we're giving it? And then maybe pass `start` and `finish` that does include `company` and make sure it gets fixed on the second go around?

Reply:
I think I've addressed your feedback @magnusvk 👍